engine/core/src/main/java/org/datacleaner/job/builder/AnalysisJobBuilder.java

```diff
@@ -1205,4 +1219,37 @@ public Boolean eval(InputColumn<?> inputColumn) {
 
         return result;
     }
+
+    public AnalysisJobBuilder getTopLevelJobBuilder(){
```

Comment:
And I would almost also expect a normal "get parent" now that we are adding these.